proj/src/components/time-tracking/TimePreview.js

```diff
@@ -0,0 +1,92 @@
+
+import React, { Component } from 'react'
+import PropTypes from 'prop-types';
+import moment from 'moment'
+import styled, { ThemeProvider } from 'styled-components'
```

Comment:
you can use ThemeProvider once in the highest component in the tree and it will be passed down to all children. App is a good place